freebsd/CMakeLists.txt

```diff
@@ -0,0 +1,38 @@
+cmake_minimum_required(VERSION 3.6.3...3.25 FATAL_ERROR)
+
+add_library(hidapi_hidraw
+    ${HIDAPI_PUBLIC_HEADERS}
+    hid.c
+)
+target_link_libraries(hidapi_hidraw PUBLIC hidapi_include)
+
+find_package(Threads REQUIRED)
+
+include(FindPkgConfig)
+pkg_check_modules(libusb-1.0 REQUIRED IMPORTED_TARGET libusb-1.0)
```

Comment:
just to double-check - is there a minimum version of libusb that is required (at least to compile/API-wise)